### PR TITLE
Remove all healthchecks that have been moved to Dockerfiles

### PR DESCRIPTION
**Pull request by adrianclay**
Moved the healthchecks to the Dockerfiles so they are more portable.  Now removing the duplication from this file.

Depends on:
https://github.com/alphagov/government-frontend/pull/618
https://github.com/alphagov/asset-manager/pull/359
https://github.com/alphagov/frontend/pull/1351
https://github.com/alphagov/publisher/pull/695
https://github.com/alphagov/collections-publisher/pull/274
https://github.com/alphagov/travel-advice-publisher/pull/251
https://github.com/alphagov/specialist-publisher/pull/1107

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -224,13 +224,16 @@ services:
       DISABLE_QUEUE_PUBLISHER: 1
       SENTRY_CURRENT_ENV: publishing-api-worker
       SENTRY_DSN: http://user:password@error-handler.dev.gov.uk/123
+    healthcheck:
```
Comment (thomasleese) [inline docker-compose.yml]: Why do we want to disable the health check having added it to the image?

Comment (adrianclay) [reply docker-compose.yml]: Ahh, forgot to mention in the commit message.  I'll update that now.  For the "worker" containers where we've overwritten the command to run ` sidekiq ` we're now disabling the healthcheck as it made no sense.  Docker was marking the container as "unhealthy", as HTTP was unreachable.

Comment (thomasleese) [reply docker-compose.yml]: Ah yep, makes sense. Wonder if disabling the worker health checks could be moved into a separate commit.

Comment (adrianclay) [reply docker-compose.yml]: Done.

**Pull request merged**